libdnf/hy-goal.c

```diff
@@ -1074,14 +1131,54 @@ hy_goal_describe_problem(HyGoal goal, unsigned i)
         }
         return g_strdup(string->str);
     }
+    pset = hy_goal_broken_dependency_all_pkgs(goal, DNF_PACKAGE_STATE_INSTALLED);
+    const int count = dnf_packageset_count(pset);
```

Comment:
it is related to this PR.... dnf_packageset_count() is declared as `unsigned`, so instead of `const int` use `unsigned`. Or make additional commit before this changes to change type to guint or something like that.